docs/02.API-REFERENCE.md

```diff
@@ -3768,6 +3768,92 @@ jerry_value_to_bigint (const jerry_value_t value);
 - [jerry_value_is_bigint](#jerry_value_is_bigint)
 - [jerry_get_bigint_digits](#jerry_get_bigint_digits)
 
+## jerry_value_as_integer
+
+**Summary**
+
+Call the ToInteger abstract operation for the given api value.
```

Comment:
This comment is still incorrect. The `jerry_value_as_integer` does not call toNumber, only works with numbers (and returns with 0 otherwise). This must be described here (similar cases below).